addons/calendar/wizard/calendar_provider_config.py

```diff
@@ -21,12 +21,18 @@ class CalendarProviderConfig(models.TransientModel):
     cal_client_secret = fields.Char(
         "Google Client_key",
         default=lambda self: self.env['ir.config_parameter'].get_param('google_calendar_client_secret'))
+    cal_sync_paused = fields.Boolean(
+        "Google Synchronization Paused",
+        default=lambda self: self.env['ir.config_parameter'].get_param('google_calendar_sync_paused'))
```

Comment:
Be aware that the system config parameter could be modified directly form the system parameter list.
If the user set it to False, it will write 'False' and the string 'False' is not falsy.
It would be better to use 
str2bool(self.env["ir.config_parameter"].sudo().get_param('google_calendar_sync_paused'), default=False)
That way, if the parameter is not set, it will be False, if it is 'False', it will be False, and True/'True' values are Ok too.

![image](https://github.com/odoo/odoo/assets/50832879/4c81d410-ddcc-4e6b-b0bb-8e2acbc7d47e)